frontend/src/components/AppNavBar.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div id="nav">
-      <AppLogo/>
+      <AppLogo :first="'S'" :last="'C'"/>
       <div class="links">
-        <router-link class="link-item" tag="h2" to="/">Home</router-link> |
-        <router-link class="link-item" tag="h2" to="/about">About</router-link>
+        <router-link class="link-item" tag="h2" to="/">Home</router-link> | 
```

Comment:
IS THAT WHITE SPACE AT THE END OF THIS LINE!?!?!?!?!

Reply:
unplayable

Reply:
Literally unplayable.